client/components/NetworkForm.vue

```diff
@@ -167,6 +167,16 @@
 					maxlength="300"
 				/>
 			</div>
+			<div class="connect-row">
+				<label for="connect:leaveMessage">Away message</label>
```

Comment:
I suppose the label "Away message" is a mistake and should probably read "Leave message"? :)

Reply:
True i'll change it